package.json

```diff
@@ -33,25 +33,28 @@
     "@semantic-release/npm": "7.1.3",
     "@semantic-release/release-notes-generator": "9.0.3",
     "@types/fs-extra": "9.0.11",
-    "@types/jest": "26.0.23",
-    "@types/jest-image-snapshot": "4.3.0",
-    "@types/puppeteer": "5.4.3",
+    "@types/jest": "29.5.11",
+    "@types/jest-image-snapshot": "6.4.0",
+    "@types/puppeteer": "7.0.4",
+    "@typescript-eslint/eslint-plugin": "^6.20.0",
+    "@typescript-eslint/parser": "^6.20.0",
     "chalk": "4.1.1",
+    "eslint": "^8.56.0",
```

Comment:
I suggest using explicit versions for all of these, instead of the `^` operator, matching the rest of them.

Reply:
Looks like one is still using it.